crypto/provider_core.c

```diff
@@ -776,11 +777,6 @@ static int core_get_params(const OSSL_PROVIDER *prov, OSSL_PARAM params[])
     int i;
     OSSL_PARAM *p;
 
-#ifndef FIPS_MODE
-    /* Load config before we attempt to read any provider parameters */
-    OPENSSL_init_crypto(OPENSSL_INIT_LOAD_CONFIG, NULL);
-#endif
```

Comment:
Side note:  I wonder why we had that in the first place...  maybe it was me, but I cannot for the life of me remember the reason.

Reply:
The lines after this use the loaded config entries... So it was to ensure they were loaded I guess. If we need to do this it needs to go elsewhere.

Reply:
I added it to ensure that the provider config file entries have been read. However, one of the calls here:

https://github.com/openssl/openssl/blob/dca97d0062397005a33dd9fd24b1238ebe9e52e7/crypto/provider_core.c#L162-L179

or here:

https://github.com/openssl/openssl/blob/dca97d0062397005a33dd9fd24b1238ebe9e52e7/crypto/provider_core.c#L625-L641

should have already been hit before we get here, so this is probably ok to go.